crates/runtime/src/datafusion/query/builder.rs

```diff
@@ -40,6 +42,7 @@ impl QueryBuilder {
             query_id: Uuid::new_v4(),
             nsql: None,
             restricted_sql_options: None,
+            protocol: Protocol::Http,
```

Comment:
We shouldn't assume this is HTTP - this should be a required parameter.

Reply:
Updated, thank you!